### PR TITLE
trailscraper 0.6.4

### PR DESCRIPTION
**Pull request by flosell**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

just released, new version of [trailscraper](https://github.com/flosell/trailscraper), resources generated using ` homebrew-pypi-poet`

### CODE REVIEW

File: Formula/trailscraper.rb [hunk 1]
```diff
@@ -3,9 +3,8 @@ class Trailscraper < Formula
 
   desc "Tool to get valuable information out of AWS CloudTrail"
   homepage "https://github.com/flosell/trailscraper"
-  url "https://github.com/flosell/trailscraper/archive/0.6.1.tar.gz"
-  sha256 "9ca2535d397282ffc09e5ca05af19bc36dd3da02235df963c3016b9f2dc4d151"
-  license "Apache-2.0"
+  url "https://github.com/flosell/trailscraper/archive/0.6.4.tar.gz"
+  sha256 "4cc20f257c8167ca0f48aab3f0f2e29e60bd8ab5316d3f16e88a35e5eab039cc"
```
Comment (claui) [inline Formula/trailscraper.rb:7]:
```suggestion
  sha256 "4cc20f257c8167ca0f48aab3f0f2e29e60bd8ab5316d3f16e88a35e5eab039cc"
  license "Apache-2.0"
```

Comment (claui) [reply Formula/trailscraper.rb:7]: I assume the `license` tag was removed in error?

The tag was only recently rolled out broadly so maybe one of the scripts you used doesn’t know this is a thing 🤔

**Review by claui — CHANGES_REQUESTED** [review]

File: Formula/trailscraper.rb (hunk 1 above)
Comment (flosell) [reply Formula/trailscraper.rb:7]: Whoops, sorry about that! Thanks for catching it! And you are right, my scripting didn't know this was a thing :) I'll fix that!

**Review by claui — APPROVED** [review]

**Comment by claui** [comment]
Thanks @flosell! 🍻

**Pull request closed without merging**